endpoints/openrtb2/auction.go

```diff
@@ -109,6 +111,7 @@ type endpointDeps struct {
 	cache                     prebid_cache_client.Client
 	debugLogRegexp            *regexp.Regexp
 	privateNetworkIPValidator iputil.IPValidator
+	storedRespFetcher         stored_requests.Fetcher
```

Comment:
Where is the code that checks for the stored response signal, and then bypasses sending a request to the bidder and just providing the stored response to the flow?

Reply:
This PR is only about stored response fetcher. 
We decided to have smaller PRs that are easy to review rather than having big and complicated PRs.
I understand this fetcher is not connected to anything and has no usages. 
I'll add actual code related to stored responses in another PR.